test/python/compiler/test_disassembler.py

```diff
@@ -211,6 +224,163 @@ def test_multiple_conditionals_multiple_registers(self):
         self.assertEqual(circuits[0], qc)
         self.assertEqual({}, header)
 
+    def test_single_circuit_calibrations(self):
+        """Test that disassembler parses single circuit QOBJ calibrations (from QOBJ-level)."""
+        theta = Parameter('theta')
+        qc = QuantumCircuit(2)
+        qc.h(0)
+        qc.append(RXGate(np.pi), [0])
+        qc.append(RXGate(theta), [1])
+        qc = qc.assign_parameters({theta: np.pi})
+
+        with pulse.build() as h_sched:
+            pulse.play(pulse.library.Drag(1, 0.15, 4, 2), pulse.DriveChannel(0))
+
+        with pulse.build() as x180:
+            pulse.play(pulse.library.Gaussian(1, 0.2, 5), pulse.DriveChannel(1))
+
+        qc.add_calibration('h', [0], h_sched)
+        qc.add_calibration(RXGate(np.pi), [0], x180)
+        qc.add_calibration(RXGate(np.pi), [1], x180)
+
+        qobj = assemble(qc, FakeOpenPulse2Q())
+        dasm_circuits, _, _ = disassemble(qobj)
```

Comment:
It's a bit challenging in all the assert statements below to quickly scan for and find the "result" that we are testing. I'd suggest doing something like this:
```suggestion
        circuits, _, _ = disassemble(qobj)
        output_circ = circuits[0]
```
if you think `output_circ` is a bit easier to parse than `dasm_circuits[0]`. No need for us to remember why we have to index into [0] each time